server/webapp/utils.py

```diff
@@ -204,6 +204,27 @@ def load_parset(project_id, parset_id, raise_exception=True):
     return _load_project_child(project_id, parset_id, ParsetsDb, ParsetDoesNotExist, raise_exception)
 
 
+def load_parset(project_id, parset_id, raise_exception=True):
```

Comment:
if you merge develop, you can refactor all those load methods and use `_load_project_child`https://github.com/optimamodel/Optima/blob/develop/server/webapp/utils.py#L170